vms/organization/models.py

```diff
@@ -1,13 +1,14 @@
-from django.core.validators import RegexValidator
-from django.db import models
-
-class Organization(models.Model):
-    name = models.CharField(
-        unique=True,            
-        max_length=75,
-        validators=[
-            RegexValidator(
-                r'^[(A-Z)|(a-z)|(0-9)|(\s)|(\-)|(:)|(\')]+$',
-            ),
-        ],
-    )
+from django.core.validators import RegexValidator
+from django.db import models
+
+class Organization(models.Model):
+    id = models.AutoField(primary_key=True)
```

Comment:
@amruthasangeeth Is there any other change in this file other than the newly added id?